config/schema.error.yaml

```
@@ -331,12 +331,18 @@
     suggestion: 'Support for the Braintree module is no longer included with Magento 2.4.0 and later. Remove the CONFIG__STORES__DEFAULT__PAYMENT__BRAINTREE__CHANNEL variable from the variables section of the .magento.app.yaml file. For Braintree support, use an official Braintree Payments extension from the Magento Marketplace instead.'
     stage: deploy
     type: critical
-!php/const Magento\MagentoCloud\App\Error::DEPLOY_WRONG_SEARCH_ENGINE:
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_ES_SERVICE_NOT_INSTALLED:
     step: validate-config
     title: 'Magento 2.4.0 requires Elasticsearch service to be installed'
     suggestion: 'Install Elasticsearch service'
     stage: deploy
     type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_WRONG_SEARCH_ENGINE:
+    step: validate-config
+    title: 'The search engine must be set to Elasticsearch for Magento >= 2.4.0'
+    suggestion: 'Check the SEARCH_CONFIGURATION variable for the `engine` option. If it is configured, remove the option, or set the value to elasticsearch.'
```

Not sure if you can use the quotes around the value `elasticsearch` , but it would be helpful to set it off as something you enter in the configuration.

```suggestion
    suggestion: 'Check the SEARCH_CONFIGURATION variable for the `engine` option. If it is configured, remove the option, or set the value to "elasticsearch".'
```

---

Updated, thank you